### PR TITLE
Structure reworked

### PR DESCRIPTION
**Pull request by FranFin**
Structure of the Intro page changed and made similar to that of IS.
Nomenclature/style uniformed.
Two images added to index.rst.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -10,24 +10,68 @@ SOSS Documentation
     :alt: eProsima
     :target: http://www.eprosima.com/
 
-:code:`soss` is a *System-Of-Systems Synthesizer* that allows communication among an arbitrary number of protocols that
+*SOSS* is a *System-Of-Systems Synthesizer* that allows communication among an arbitrary number of protocols that
 speak different languages.
 
 If one has, for instance, a pair of complex systems, A and B, and wills to combine them to create a larger, even more
-complex system, :code:`soss` can be chosen as an intermediate message-passing tool that, by speaking a common
+complex system, *SOSS* can be chosen as an intermediate message-passing tool that, by speaking a common
```
Comment (LuisGP) [inline docs/index.rst]: I don't like the `can be chosen` expression. I'd change it to `allows` or something similar.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -86,30 +85,30 @@ how messages should be passed among the middlewares involved:
...
 
 **Note**: the sourcing of the local colcon overlay is required every time the colcon workspace is opened in
 a new shell environment.
 As an alternative, you can copy the source command with the full path of your local installation to your 
-:code:`.bashrc` file. For instance, for :code:`soss` you would add:
+:code:`.bashrc` file. For instance, for *SOSS* you would add:
 
 .. code-block:: bash
 
     source ~/PATH_TO_WORKSPACE/workspaces/soss/install/setup.bash
```
Comment (LuisGP) [inline docs/getting_started.rst]:
```suggestion
    source PATH_TO_WORKSPACE/workspaces/soss/install/setup.bash
```

**Review by LuisGP — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 1 above)
Comment (FranFin) [reply docs/index.rst]: ok, lo intento reformular.

File: docs/getting_started.rst (hunk 2 above)
Comment (FranFin) [reply docs/getting_started.rst]: ok.

**Review by LuisGP — APPROVED** [review]

File: docs/index.rst [hunk 3]
```diff
@@ -15,15 +15,17 @@ speak different languages.
 
 If one has a number of complex systems and wills to combine them to create a larger, even more
 complex system, *SOSS* can act as an intermediate message-passing tool that, by speaking a common
-language, mediates the integration. The inter-systems bridges are centralized and configured with a YAML text file,
-through which the user can provide a mapping between the topics and services on the middlewares of the involved systems.
+language, centralizes and mediates the integration.
```
Comment (LuisGP) [inline docs/index.rst:18]: I miss the introduction to the YAML files, and the mapping between the topics and services.

File: docs/index.rst [hunk 4]
```diff
@@ -15,15 +15,17 @@ speak different languages.
...
 
-The communication between the different protocols is made possible by system-specific plugins, that we will refer to
-henceforth as **System-Handles**.
-These provide the necessary conversion between the target protocol and the specific language
+The communication between the different protocols is made possible by middleware-specific plugins, or
```
Comment (LuisGP) [inline docs/index.rst]: I strongly prefer `system-specific`.

**Review by LuisGP — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 3 above)
Comment (FranFin) [reply docs/index.rst:18]: They are below!

File: docs/index.rst (hunk 4 above)
Comment (FranFin) [reply docs/index.rst]: ok.

File: docs/index.rst (hunk 3 above)
Comment (LuisGP) [reply docs/index.rst:18]: My fault.

File: docs/index.rst (hunk 4 above)
Comment (LuisGP) [reply docs/index.rst]: Tx

**Review by LuisGP — APPROVED** [review]

**Pull request merged**